pkg/radish/radish.go

```diff
@@ -43,11 +50,72 @@ func RunRadish(args []string) {
 	os.Exit(exitCode)
 }
 
+//RunNodeJS :
+func RunNodeJS(mainJavaScriptFile string, logLocation string, logFilename string, logFileSize int) {
+	e := nodejs.NewNodeJSExecutor()
+
+	cmd := e.PrepareForNodeJSRun(mainJavaScriptFile)
+
+	writer := logw.NewLogWriter(logw.WithLogLocation(logLocation), logw.WithFilename(logFilename), logw.WithWriteToFile(true), logw.WithRotateSize(logFileSize))
+
+	stdoutPipe, pipeerr := cmd.StdoutPipe()
+	if pipeerr != nil {
+		logrus.Fatalf("Something wrong with stdout pipe: %s", pipeerr)
+	}
+
+	stderrPipe, pipeerr := cmd.StderrPipe()
+	if pipeerr != nil {
+		logrus.Fatalf("Something wrong with stderr pipe: %s", pipeerr)
+	}
+
+	go func() {
```

Comment:
Kan denne funksjonen feile, for eksempel under logg-rotering? Vil ikke da applikasjonen kjøre videre, uten logging?

Reply:
innspill @bj0rn ?

Reply:
Om log writeren feiler vil den logge til stderr, men ikke feile. Den vil bare prøve igjen